fastlane/lanes/release.rb

```diff
@@ -1,21 +1,15 @@
 # frozen_string_literal: true
 
 platform :android do
-  #####################################################################################
-  # code_freeze
-  # -----------------------------------------------------------------------------------
-  # This lane executes the steps planned on code freeze
-  # -----------------------------------------------------------------------------------
-  # Usage:
-  # bundle exec fastlane code_freeze [update_release_branch_version:<update flag>] [skip_confirm:<skip confirm>]
+  # Creates a new release branch from the current trunk for code freeze.
   #
-  # Example:
-  # bundle exec fastlane code_freeze
-  # bundle exec fastlane code_freeze update_release_branch_version:false
-  # bundle exec fastlane code_freeze skip_confirm:true
-  #####################################################################################
-  desc 'Creates a new release branch from the current trunk'
-  lane :code_freeze do |options|
+  # @param [Boolean] skip_confirm Whether to skip the confirmation prompt
+  #
+  # @example
+  #   bundle exec fastlane code_freeze
+  #   bundle exec fastlane code_freeze skip_confirm:true
+  #
```

Comment:
I know in other repos and in other places here we kept comment, converting them to this Yard-like format. So this is consistent with the rest of the repo and should stay.

But personally, I find the signal-noise ratio very low and would be fine removing a comment like this, saving it only for lanes with quirky behavior or quirky parameters that need extra explanation.

Reply:
Agreed. Those `@example` are not adding any value here (and neither in most other lane docs where we also have kept `@example`) and are more noise than helpful. I'd vote to remove them as well.

Reply:
> I find the signal-noise ratio very low and would be fine removing a comment like this

@mokagio do you mean the entire comment (description, parameters) or just the `@example` part?


---

> Those @example are not adding any value here (and neither in most other lane docs where we also have kept @example)

I see the `@example` being useful _sometimes_, but that's more of an exception 👍 
Specially if we're talking about these lanes, I agree it's not really helpful. I'll it clean up on this PR and will update other repos as I move.

Reply:
Updated on c2fac33b9dfe7fa4c6c8e1e6112b611819211ccf

Reply:
I was referring to the whole comment.

`lane :start_code_freeze do |skip_confirm: false|` is pretty clear to me. Granted, I might be victim of the [curse of knowledge](https://en.wikipedia.org/wiki/Curse_of_knowledge) given how long I've been using the tool...